app/policies/medical_shift_policy.rb

```diff
@@ -15,4 +15,8 @@ def create?
   def update?
     user.present? && record.user == user
   end
+
+  def destroy?
+    user.present?
```

Comment:
@philipeleandro , looks like we have a sec vulnerability here, please check if the same issue happens on other policies, if they do, please create an issue for them.

Also add specs for `medical_shift_policy` or test them at request level

Reply:
These other policies `HealthInsurancePolicy` and `UserPolicy`. Procedures, MedicalShift, Patient
 and Hospital Policies have the same validation by checking user presence

Reply:
I added tests for request and police 43daa84a704a3fa90e91cf6d9ad30deb871b9adc

Reply:
> These other policies HealthInsurancePolicy and UserPolicy. Procedures, MedicalShift, Patient
and Hospital Policies have the same validation by checking user presence

This is not good answer, if things are incorrect at somewhere else, this does not means that we should keep doing it.
We need to check if the user is owner of the medical_shift. Otherway User a can delete medical shift from user b

Reply:
> I added tests for request and police https://github.com/espoo-dev/med_system_backend/commit/43daa84a704a3fa90e91cf6d9ad30deb871b9adc

When I say 

> Also add specs for medical_shift_policy or test them at request level

I mean add specs `medical_shift_policy_spec`

Reply:
I know, it was add in this commit ee8ad3ca88bfef2681bbb59c1d2f8447ed08383c. I pasted a wrong hash commit KKKKKK

Reply:
I pasted a wrong hash commit. This ee8ad3ca88bfef2681bbb59c1d2f8447ed08383c is the correct one